partner_view_360/i18n/sv_SE.po

```diff
@@ -222,30 +205,36 @@ msgstr "Aktuell sedan"
 msgid "Customers"
 msgstr "Kunder"
 
+#. module: partner_view_360
+#: model:ir.model.fields,field_description:partner_view_360.field_res_partner__deactualization_date
+#: model:ir.model.fields,field_description:partner_view_360.field_res_users__deactualization_date
+#: model_terms:ir.ui.view,arch_db:partner_view_360.view_partner_form
+msgid "Deactualization date"
+msgstr "Avaktualiserad"
+
+#. module: partner_view_360
+#: model:ir.model.fields,field_description:partner_view_360.field_res_partner__deactualization_reason
+#: model:ir.model.fields,field_description:partner_view_360.field_res_users__deactualization_reason
+msgid "Deactualization reason"
+msgstr "Avaktualisering orsak"
```

Comment:
Missing s and i believe it should be a compounded word. (Skrivihopp)

Reply:
yes it should be 'Avaktualiseringsorsak' :)